components/dashboard/src/App.tsx

```diff
@@ -165,7 +169,22 @@ function App() {
         (async () => {
             var user: User | undefined;
             try {
-                const teamsPromise = getGitpodService().server.getTeams();
+                let teamsPromise: Promise<Team[]>;
+                if (usePublicApiTeamsService) {
+                    teamsPromise = teamsService.listTeams({}).then((response) => {
+                        return response.teams.map((team) => {
+                            const t: Team = {
+                                id: team.id,
+                                name: team.name,
+                                slug: team.slug,
+                                creationTime: "",
```

Comment:
As we add more public APIs does it make sense to move these RPC type -> gitpod protocol type mappings to one place or are you happy to do them inline like this?

Reply:
I'll eventually add a mapper which does the conversion. For this single use, I felt it's okay to just inline it and then pull out.